docs/resources.js

```diff
@@ -18,8 +18,13 @@ function getDisaster() {
   if (localStorage.getItem('disaster')) {
     return localStorage.getItem('disaster');
   }
-  localStorage.setItem('disaster', default_disaster);
-  return default_disaster;
+  localStorage.setItem('disaster', defaultDisaster);
+  return defaultDisaster;
+}
+
+/** @return {string} EE asset path for score asset for the current disaster */
+function getScoreAsset() {
+  return gdEePathPrefix + getDisaster() + '/data-ms-as-nod';
```

Comment:
this file name is now potentially bogus right? as in could be without any damage info at all. Not worth rerunning scripts but maybe worth leaving a note until it's changed.

Reply:
import_data.js still will only generate this asset with damage, so I don't think it's necessary yet?

Also, I didn't parse "ms-as-nod" as "Microsoft-as-no-damage" until just now (thanks!), so may not be essential.

Reply:
Agreed, actually rerunning this scripts not worth it and maybe fine if you're going to change it tomorrow but it might be nice to leave a note here indicating that what's returned by this method and what the asset path describes could be two different things.